src/handlers/sdp/MediaSection.ts

```diff
@@ -745,6 +791,18 @@ export class OfferMediaSection extends MediaSection
 		const rtxSsrc = (encoding.rtx && encoding.rtx.ssrc)
 			? encoding.rtx.ssrc
 			: undefined;
+		const payloads = offerRtpParameters!.codecs
+			.map((codec: RtpCodecParameters) => codec.payloadType);
+
+		this._mediaObject.payloads = this._mediaObject.payloads.split(' ')
+			.filter((payload: any) => !payloads.includes(Number(payload)))
+			.join(' ');
+
+		this._mediaObject.rtp = this._mediaObject.rtp.filter((rtp: any) => !payloads.includes(rtp.payload));
+
+		this._mediaObject.rtcpFb = this._mediaObject.rtcpFb.filter((rtcpFb: any) => !payloads.includes(rtcpFb.payload));
+
+		this._mediaObject.fmtp = this._mediaObject.fmtp.filter((fmtp: any) => !payloads.includes(fmtp.payload));
```

Comment:
Why those additions? I need some rationale here.

Reply:
I'm removing from the SDP the description of the payloads being stopped. This way, if those same payloads are sent again (imagine webcam stop/start), the corresponding description of the new payloads will be added again, representing exactly what is being received at each time.

I'm looking for the standard way to do this, but PlanB is not even a standard. Any comment on how we should be doing this?

Reply:
understood